egs/librispeech/asr/simple_v1/train.py

```diff
@@ -219,29 +220,18 @@ def main():
     fix_random_seed(42)
     # load L, G, symbol_table
     lang_dir = Path('data/lang_nosp')
-    symbol_table = k2.SymbolTable.from_file(lang_dir / 'words.txt')
+    lexicon_txt = lang_dir / 'lexicon.txt'
+    phones_txt = lang_dir / 'phones.txt'
+    words_txt = lang_dir / 'words.txt'
 
-    print("Loading L.fst")
-    if os.path.exists(lang_dir / 'Linv.pt'):
-        L_inv = k2.Fsa.from_dict(torch.load(lang_dir / 'Linv.pt'))
-    else:
-        with open(lang_dir / 'L.fst.txt') as f:
-            L = k2.Fsa.from_openfst(f.read(), acceptor=False)
-            L_inv = k2.arc_sort(L.invert_())
-            torch.save(L_inv.as_dict(), lang_dir / 'Linv.pt')
-
-    phone_symbol_table = k2.SymbolTable.from_file(lang_dir / 'phones.txt')
-
-    graph_compiler = TrainingGraphCompiler(
-        L_inv=L_inv,
-        vocab=symbol_table,
-    )
+    graph_compiler = CtcTrainingGraphCompiler(lexicon_txt=str(lexicon_txt),
```

Comment:
how about just `lexicon`, `phones` and `words` (or `vocab`/`vocabulary`)? "txt" is just a disk storage detail, not relevant to the concept.

Reply:
Pass the `Path` object in and use that to read it inside, e.g. there's a nice method `path.read_text()`.

Reply:
Resolved by removal.